TesseractOCR/G8Tesseract.mm

```diff
@@ -785,7 +785,12 @@ - (NSData *)recognizedPDFForImages:(NSArray*)images {
 
 - (UIImage *)imageWithBlocks:(NSArray *)blocks drawText:(BOOL)drawText thresholded:(BOOL)thresholded
 {
-    UIImage *image = thresholded ? self.thresholdedImage : self.image;
+    UIImage *image = self.thresholdedImage;
+    if (!thresholded) {
+        // Have to rotate the original image to the right or it shows upside down
+        CGImageRef imageRef = self.image.CGImage;
+        image = [[UIImage alloc] initWithCGImage:imageRef scale:self.image.scale orientation:UIImageOrientationRight];
```

Comment:
That's not a universal PR. It only works in your case. What if the image is rotated to a different orientation?